modules/lib/src/main/resources/assets/js/app/inputtype/ui/text/dialog/LinkModalDialog.ts

```diff
@@ -64,13 +65,17 @@ export class LinkModalDialog
     private textFormItem: FormItem;
     private toolTipFormItem: FormItem;
     private mediaOptionRadioFormItem: FormItem;
+    private anchorFormItem: FormItem;
+    private paramsFormItem: FormItem;
     private protocolsDropdownButton: MenuButton;
 
     private contentId: ContentId;
     private parentSitePath: string;
 
     private tabNames: any;
 
+    private paramsFormIds: {keyId: string, valueId: string}[];
```

Comment:
Use [Map](https://howtodoinjava.com/typescript/maps/) instead. This will allow you use `.set`, `.delete` etc.